site/gdocs/ArticleBlock.tsx

```diff
@@ -587,6 +587,34 @@ export default function ArticleBlock({
                 {...block}
             />
         ))
+        .with({ type: "blockquote" }, (block) => {
+            const isCitationAUrl = Boolean(
+                block.citation && block.citation.startsWith("http")
+            )
+            const blockquoteProps = isCitationAUrl
+                ? { cite: block.citation }
+                : {}
+            return (
+                <blockquote
+                    className={cx(getLayout("blockquote", containerType))}
+                    {...blockquoteProps}
+                >
+                    {block.text.map((textBlock, i) => (
+                        <Paragraph
+                            className="article-block__text"
+                            d={textBlock}
+                            key={i}
+                        />
+                    ))}
+
+                    {isCitationAUrl ? null : (
+                        <footer>
+                            <cite>{block.citation}</cite>
+                        </footer>
+                    )}
```

Comment:
If `citation` is truly optional, this might produce an empty `<footer><cite></cite></footer>` (although it doesn't at the moment because it errors out if no citation is given)